tests/test_repository.py

```diff
@@ -10,24 +10,24 @@
 def test_repository_not_zero_commits():
     """Check to ensure that GatorGrader's repository registers."""
     commits = repository.get_commits(".")
-    assert len(commits) > 1
+    assert len(commits) > 0
 
 
 def test_repository_not_zero_commits_extra_method():
     """Check to ensure that GatorGrader's repository registers."""
     commits = repository.get_commits(".")
-    assert repository.count_commits(commits) > 1
+    assert repository.count_commits(commits) > 0
 
 
 def test_repository_not_zero_commits_greater_than():
     """Check to ensure that commit counts work correctly."""
-    valid, __ = repository.commits_greater_than_count(".", 1)
+    valid, __ = repository.commits_greater_than_count(".", 0)
     assert valid is True
 
 
 def test_repository_not_zero_commits_greater_than_exacted():
```

Comment:
I agree that it is fine to make these changes to the test cases --- they still meet the overall goals of the tests. With that said, I am curious: what prompted you to change these tests as a part of this PR, @Michionlion?

Reply:
The `checkout` action only clones one commit by default, and I wanted to keep it that way.